specs/Media.md

```diff
@@ -0,0 +1,146 @@
+# Background
+WebView2 is a control that developers can implement for part of, or for the entirety of their app. In the either case, developers may need the ability to control the media within the webview's bounds. This API will allow developers to control the sound of each webview2 instance by enabling the mute/unmute of the media within the webview. There will also be the ability to check if the sound is muted or audible.
+
+# Description
+Enable end developer to `Mute`, `Unmute` a webview content. And ability to check for the `IsMuted` and `IsCurrentlyAudible`.
+
+# Examples
+
+The following code snippet demonstrates how the Media related API can be used:
+
+## Win32 C++
+
+```cpp
+bool ViewComponent::HandleWindowMessage(
+    HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam, LRESULT* result)
+{
+    if (message == WM_COMMAND)
+    {
+        case IDM_MUTE:
+            m_webView->Mute();
+            return true;
+        case IDM_UNMUTE:
+            m_webView->Unmute();
+            return true;
+        case IDM_IS_MUTED:
+            BOOL isMuted;
+            m_webView->get_IsMuted(&isMuted);
+            MessageBox(nullptr, isMuted ? L"Yes" : L"No", L"Is Muted", MB_OK);
+            return true;
+        case IDM_IS_CURRENTLY_AUDIBLE:
```

Comment:
We'd probably want to demonstrate the events for the muted and audible properties. Like putting an icon at the end of the sample app's window's title when the values change like 🔇 for muted and 🔊 for audible.

Reply:
Make sense. I will make the change to code and also this spec

Reply:
updated a bit. Still debating on whether to use the IsMutedChanged event handler